source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2860,6 +2966,10 @@ export class Container extends CardElement {
             invokeSetParent(this._selectAction, this);
         }
     }
+
+    get items(): Array<CardElement> {
```

Comment:
This can't exist. If we introduce this, developers will use it to add new elements, which they can't do because that doesn't properly set the added element's parent. Developers are supposed to use addItem to add elements. Please remove.